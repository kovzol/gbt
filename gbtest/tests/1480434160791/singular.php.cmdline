#!/bin/sh
cd tests/1480434160791
cat singular.php.in | /usr/bin/timeout -k 120 60 /usr/bin/time /usr/bin/Singular > singular.php.out 2>singular.php.err