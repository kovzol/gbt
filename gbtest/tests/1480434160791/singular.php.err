0.00user 0.00system 0:00.00elapsed 57%CPU (0avgtext+0avgdata 10084maxresident)k
0inputs+8outputs (0major+410minor)pagefaults 0swaps
