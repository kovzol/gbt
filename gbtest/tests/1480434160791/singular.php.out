                     SINGULAR                                 /
 A Computer Algebra System for Polynomial Computations       /   version 4.0.3
                                                           0<
 by: W. Decker, G.-M. Greuel, G. Pfister, H. Schoenemann     \   Jan 2016
FB Mathematik der Universitaet, D-67653 Kaiserslautern        \  Debian 4.0.3+ds-1
_[1]=v1
_[2]=v2
_[3]=v3
_[4]=v4
_[5]=v5-1
_[6]=v6
_[7]=v7
_[8]=v8
_[9]=v9
_[10]=v10-1
_[11]=v17-v15
_[12]=v19-v15
_[13]=v18-v16-1
_[14]=v20
_[15]=v16*v13-v15*v14
_[16]=v16^2+v15^2-1
_[17]=v23^2-1
_[18]=v25^2-v15^2+2*v15-1
_[19]=v24^2+2*v15*v13-v13^2+2*v16*v14-v14^2-1
_[20]=v26^2-v15^2
_[21]=2*v15^2*v14-2*v15*v13*v14-v16*v14^2+2*v16*v15-4*v15*v14+2*v13*v14-v16+2*v14
_[22]=2*v16*v15*v14-v15*v14^2-2*v15^2-2*v16*v14+v14^2-v15+1
_[23]=2*v15*v13^2+2*v15*v14^2+4*v15*v13-2*v13^2+2*v16*v14-v14^2+2*v15-4*v13-1
_[24]=2*v15^2*v13+v15*v14^2+2*v15^2+2*v16*v14-v14^2+v15-2*v13-1
_[25]=2*v16*v14^3-v14^4-2*v15*v14^2-4*v15*v13+4*v13^2-2*v16*v14-2*v15+4*v13+1
_[26]=2*v15*v13*v14^2+v14^4+6*v15*v14^2-2*v13*v14^2+2*v15*v13-4*v13^2+4*v16*v14-4*v14^2+2*v15-6*v13-1
_[27]=2*v15*v14^4-v13*v14^4+v14^4+4*v13^3+8*v15*v14^2-2*v13*v14^2+8*v15*v13-4*v13^2+8*v16*v14-6*v14^2+6*v15-13*v13-3
_[28]=v13^2*v14^4+v14^6-4*v13^4-2*v13^2*v14^2-2*v14^4-8*v13^3-3*v13^2+v14^2
> Auf Wiedersehen.
